db-demo/src_hd/db/demos/PlayerInterface/PlayerInterface.xhtml

```diff
@@ -0,0 +1,69 @@
+<html xmlns="http://www.w3.org/1999/xhtml"
+	xmlns:f="http://xmlns.jcp.org/jsf/core"
+	xmlns:h="http://xmlns.jcp.org/jsf/html"
+	xmlns:ui="http://xmlns.jcp.org/jsf/facelets"
+	xmlns:ic="http://ivyteam.ch/jsf/component"
+	xmlns:p="http://primefaces.org/ui"
+	xmlns:pe="http://primefaces.org/ui/extensions">
+<h:body>
+	<ui:composition template="/layouts/frame-10.xhtml">
+		<ui:define name="title">PlayerInterface</ui:define>
+		<ui:define name="content">
+
+
+			<h:form id="form1">
```

Comment:
this 'form1' id is referenced in update statement: so I'd use some more telling ... .e.g 'playerListForm'